### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by IIILIGHTIII**
https://IIILIGHTIII.github.io/layout_product-cards/
https://IIILIGHTIII.github.io/layout_product-cards/report/html_report/

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -5,9 +5,60 @@
...
+    <link rel="stylesheet" href="./styles/style.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="prod-card" data-qa="card">
+
+      <img src="./images/imac.jpeg"
+      alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+      class="prod-card__img"
+      >
```
Comment (IvanFesenko) [inline src/index.html:22]: Pay attention on code style
```suggestion
      <img 
        src="./images/imac.jpeg"
        alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
        class="prod-card__img"
      >
```

File: src/index.html [hunk 2]
```diff
@@ -5,9 +5,60 @@
...
+      <p class="prod-card__price">
+        <span class="text">Price</span>
+
+        <span class="cost">$2,199</span>
+      </p>
+
+      <a href="#"
+      class="prod-card__buy"
+      data-qa="hover"
+      >
```
Comment (IvanFesenko) [inline src/index.html:59]: Same here
```suggestion
      <a 
        href="#"
        class="prod-card__buy"
        data-qa="hover"
      >
```

File: src/styles/prod-card.scss [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+}
+
+.prod-card {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  font-family: "Roboto", Helvetica, sans-serif;
```
Comment (IvanFesenko) [inline src/styles/prod-card.scss:12]: move `font-family` into `html` selector

File: src/styles/prod-card.scss [hunk 4]
```diff
@@ -0,0 +1,109 @@
+* {
+  margin: 0;
+}
+
+.prod-card {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  box-sizing: border-box;
```
Comment (IvanFesenko) [inline src/styles/prod-card.scss:9]: move `box-sizing: border-box;` into * selector

File: src/styles/prod-card.scss [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+    line-height: 14px;
+    margin-bottom: 16px;
+    color: #{$light-gray};
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    box-sizing: border-box;
```
Comment (IvanFesenko) [inline src/styles/prod-card.scss:42]:
```suggestion

```

**Review by IvanFesenko — CHANGES_REQUESTED** [review]
Check my comments

File: src/index.html [hunk 6]
```diff
@@ -5,9 +5,62 @@
...
   <body>
-    <h1>Product cards</h1>
+    <div class="prod-card" data-qa="card">
+
+      <img
+      src="./images/imac.jpeg"
+      alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+      class="prod-card__img"
+      >
+
```
Comment (IvanFesenko) [inline src/index.html:24]: Indents not fixed
```suggestion
      <img
        src="./images/imac.jpeg"
        alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
        class="prod-card__img"
      >

```

File: src/index.html [hunk 7]
```diff
@@ -5,9 +5,62 @@
...
+      <p class="prod-card__price">
+        <span class="text">Price</span>
+
+        <span class="cost">$2,199</span>
+      </p>
+
+      <a
+      href="#"
+      class="prod-card__buy"
+      data-qa="hover"
```
Comment (IvanFesenko) [inline src/index.html:60]: same here

**Review by IvanFesenko — CHANGES_REQUESTED** [review]
Not fully fix previous comments

**Review by lerastarynets — APPROVED** [review]
lgtm 🚀

**Pull request closed without merging**